HeterogeneousCore/AlpakaInterface/interface/HistoContainer.h

```diff
@@ -27,10 +27,10 @@ namespace cms::alpakatools {
       const uint32_t nt = offsets[nh];
       for (uint32_t i : elements_with_stride(acc, nt)) {
         auto off = alpaka_std::upper_bound(offsets, offsets + nh + 1, i);
-        ALPAKA_ASSERT_OFFLOAD((*off) > 0);
+        ALPAKA_ASSERT_ACC((*off) > 0);
```

Comment:
So in order to replicate the behavior of `assert()` + `HeterogeneousCore/CUDAUtilities/interface/cuda_assert.h` we'd want the `ALPAKA_DISABLE_ASSERT_ACC` to be defined by default?

Reply:
Yes, I think so.

Reply:
I'm thinking to keep `ALPAKA_ASSERT_ACC` enabled on the GPU devices for the time being, while we integrate and test the various alpaka-based code - since `ALPAKA_ASSERT_OFFLOAD` was also enabled - and later disable it centrally adding `-DALPAKA_DISABLE_ASSERT_ACC` to the alpaka tool file.

Reply:
Makes sense. Should we start adding e.g.
```xml
<flags REM_CXXFLAGS="-DALPAKA_DISABLE_ASSERT_ACC"/>
```
to all the unit tests that make use of `ALPAKA_ASSERT_ACC`?

Reply:
That sounds bothersome... maybe there is some way to define (or not) a flag for all `.../test` directories ?